plugins/buildingplan.cpp

```diff
@@ -911,26 +911,75 @@ IMPLEMENT_VMETHOD_INTERPOSE(buildingplan_query_hook, render);
 IMPLEMENT_VMETHOD_INTERPOSE(buildingplan_place_hook, render);
 IMPLEMENT_VMETHOD_INTERPOSE(buildingplan_room_hook, render);
 
+DFHACK_PLUGIN_IS_ENABLED(is_enabled);
+
+static void setSetting(std::string name, bool value);
+
+static bool isTrue(std::string val)
+{
+    return val == "on" || val == "true" || val == "y" || val == "yes";
```

Comment:
Some commands accept `1` and `0` as well.

Since you're already processing the value passed in here with `toLower` on the caller's side, I might suggest moving that here, since I assume you don't ever want `YES` to be treated as false. (Unlikely, sure, but since I'm already suggesting things...)

Reply:
done and done.